src/server/views/layout.html

```diff
@@ -36,18 +36,19 @@
 
 {% block beforeContent %}
     {% if phaseTag %}
-        {% if feedbackLink %}
+        {% if 'mailto:' in feedbackLink %}
             {{ govukPhaseBanner({
                 tag: {
                     text: phaseTag | capitalize
                 },
-                html: 'This is a new service – your <a class="govuk-link" href=' + feedbackLink + '>feedback</a> will help us to improve it.'
+                html: 'This is a new service. Help us improve it and <a class="govuk-link" href="' + feedbackLink + '">give your feedback by email</a>.'
```

Comment:
Can you double check URL encoding?

Mailto links will need `&` → `&amp;` etc

```console
FEEDBACK_LINK=mailto:hello@example.org?subject=Feedback&body=Hello
```

```suggestion
                html: 'This is a new service. Help us improve it and <a class="govuk-link" href="' + feedbackLink | urlencode + '">give your feedback by email</a>.'
```

Reply:
Fixed in the following two commits.

Reply:
@alexluckett Escaping is working great, nice work 🙌